docs-gen/Makefile

```diff
@@ -15,3 +15,6 @@ clean:
 
 serve:
 	hugo server -b http://localhost:1313/
+
+install-npm-dependencies:
+	npm install
```

Comment:
This is to ensure postcss is installed (which is already in `package.json`). Without this we get the following error:
```
Error: error building site: POSTCSS: failed to transform "/scss/main.css" (text/css). You need to install PostCSS. See https://gohugo.io/functions/css/postcss/: binary with name "postcss" not found using npx
```